grpc/flatbuffers-js-grpc/greeter_grpc.js

```diff
@@ -0,0 +1,51 @@
+// Generated GRPC code for FlatBuffers JS!
+'use strict';
+var grpc = require('grpc');
+var flatbuffers = require('flatbuffers').flatbuffers;
+
+var Greeter_fbs = require('./greeter_generated.js');
+
+function serialize_ByteBuffer(buffer) {
+  if (!(buffer instanceof Uint8Array)) {
+    throw new Error('Expected argument of type ByteBuffer');
+  }
+  return new Buffer.from(buffer);
+}
+
+function deserialize_HelloReply(buffer_arg) {
+  return Greeter_fbs.HelloReply.getRootAsHelloReply(new flatbuffers.ByteBuffer(new Uint8Array(buffer_arg)), 0)
+}
+
+function deserialize_HelloRequest(buffer_arg) {
+  return Greeter_fbs.HelloRequest.getRootAsHelloRequest(new flatbuffers.ByteBuffer(new Uint8Array(buffer_arg)), 0)
+}
+
+function deserialize_ManyHellosRequest(buffer_arg) {
+  return Greeter_fbs.ManyHellosRequest.getRootAsManyHellosRequest(new flatbuffers.ByteBuffer(new Uint8Array(buffer_arg)), 0)
+}
+
+var GreeterService = exports.GreeterService = {
+  SayHello: {
+    path: '/Greeter/SayHello',
+    requestStream: false,
+    responseStream: false,
+    requestType: flatbuffers.ByteBuffer,
+    responseType: Greeter_fbs.HelloReply,
+    requestSerialize: serialize_ByteBuffer,
+    requestDeserialize: deserialize_HelloRequest,
+    responseSerialize: serialize_ByteBuffer,
+    responseDeserialize: deserialize_HelloReply,
+  },
+  SayManyHellos: {
+    path: '/Greeter/SayManyHellos',
+    requestStream: false,
+    responseStream: true,
+    requestType: flatbuffers.ByteBuffer,
```

Comment:
Wouldn't this be the client stream and bidirectional stream? (I just took the swift greeter.fbs and made It here)

Reply:
Do you have an example for this `exception handling`?

Reply:
I was perusing the python grpc API (I'm not familiar with the Swift API.) There is a good deal of stuff there that doesn't seem to be here -- at least on a quick look -- but having a minimal implementation would be great.

Wrt to streaming in general -- you have the option for streaming but it seems that the `ManyHellosRequest` blocks until the whole array is passed.

Reply:
I honestly took this for a reference, so I would assume this is how it would be in nodeJS
- https://github.com/grpc/grpc/blob/v1.32.0/examples/node/static_codegen/helloworld_grpc_pb.js
- https://github.com/grpc/grpc/blob/v1.32.0/examples/node/static_codegen/greeter_server.js

Reply:
However, you do say that the reply type is`responseType: Greeter_fbs.HelloReply` for `SayManyHellos` service, which would indeed imply a stream.